sql/src/main/java/io/crate/execution/dsl/projection/SourceIndexWriterProjection.java

```diff
@@ -178,9 +202,44 @@ public void writeTo(StreamOutput out) throws IOException {
                 out.writeString(exclude);
             }
         }
+        Symbols.toStream(outputs, out);
     }
 
     public boolean overwriteDuplicates() {
         return overwriteDuplicates;
     }
+
+    @Nullable
+    public ReturnSummarySymbols returnSummarySymbols() {
+        return returnSummarySymbols;
+    }
+
+    public static class ReturnSummarySymbols {
+        private final InputColumn sourceUriSymbol;
```

Comment:
Minor: Given that there are no `sourceUri` other than `sourceUriSymbol` it might make sense to drop the `Symbols` suffix. There is already the type info and the container class is also called `ReturnSummarySymbols`.